examples/website/safegraph/app.js

```diff
@@ -23,6 +23,7 @@ const colorScale = scaleLog()
 export function renderToDOM(container, data) {
   const map = new mapboxgl.Map({
     container,
+    useWebGL2: true,
```

Comment:
Just curious why this is required? Do we require that mapbox and deckgl use the same WebGL version?

Reply:
They are using the same context, and luma won't work with WebGL1